### ISSUE TITLE
SyntaxError: Unexpected token (1:0)

### ISSUE THREAD

**Issue by goodbomb** [styled-components/stylelint-processor-styled-components#125]
Hi,

I'm getting this error when trying to use this processor with webpack:

```
SyntaxError: Unexpected token (1:0)
    at Parser.pp$5.raise (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:4454:13)
    at Parser.pp.unexpected (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:1761:8)
    at Parser.pp$3.parseExprAtom (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:3750:12)
    at Parser.parseExprAtom (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:7238:22)
    at Parser.pp$3.parseExprSubscripts (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:3494:19)
    at Parser.pp$3.parseMaybeUnary (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:3474:19)
    at Parser.pp$3.parseExprOps (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:3404:19)
    at Parser.pp$3.parseMaybeConditional (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:3381:19)
    at Parser.pp$3.parseMaybeAssign (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:3344:19)
    at Parser.parseMaybeAssign (.../node_modules/stylelint-processor-styled-components/node_modules/babylon/lib/index.js:6474:20)
```

Not sure what to do about it, but I'm using the following:

NodeJS: 8.4.0
Webpack: 3.5.6
style-loader: 0.18.2
stylelint: 8.1.1
stylelint-config-standard: 17.0.0
stylelint-config-styled-components: 0.1.1
stylelint-custom-processor-loader: 0.2.1
stylelint-processor-styled-components: 0.4.0
stylelint-webpack-plugin: 0.9.0

My `.stylelintrc` has the following as per the docs:

```
{
    "processors": ["stylelint-processor-styled-components"],
    "extends": [
        "stylelint-config-standard"
    ],
    "syntax": "scss",
    "rules": {...}
}
```

Any help would be appreciated.

**Comment by emilgoldsmith** [comment]
Hey @goodbomb!

If you're using the `stylelint-webpack-plugin` it will probably break stuff as if I remember correctly when I tested it before creating `stylelint-custom-processor-loader` because of these problems, which I see you are also using, it simply takes only and all css, so if that's your stylelint config it'll give our processor css and we assume it's JS, therefore an obvious syntax error.

The bottom line is you can only pass JS to this processor.

If you rule out this as the problem please share your webpack config + try to narrow down the error to a single file and show us that file

**Comment by goodbomb** [comment]
Ah! Perfect. That did the trick. Thanks!
